.nancy-ignore

```diff
@@ -0,0 +1,2 @@
+# pkg:golang/golang.org/x/net@v0.32.0
+CVE-2024-45338 until=2025-03-01
```

Comment:
I would rather that we upgrade this package

![image](https://github.com/user-attachments/assets/a188a6de-e60f-4678-9169-6bef91af6713)

Reply:
wait, wasn't renovate supposed to do this if an update was available?? :thinking:

Reply:
maybe but it's not always doing its job properly as it seems

Reply:
=> updating all deps here: https://github.com/giantswarm/observability-operator/pull/215

Reply:
updating this dependency alone would have been enough for the CVE fix